generators/entity-client/templates/react/src/main/webapp/app/entities/entity-dialog.tsx.ejs

```diff
@@ -256,6 +256,15 @@ _%>
                 </option>
               <%_ } _%>
               </AvInput>
+            <%_ } else if (fieldType === 'Integer' || fieldType === 'Long' || fieldType === 'Float' || fieldType === 'Double') { _%>
```

Comment:
use includes it will be much nicer to read
`if (['Integer', 'Long', 'Float', 'Double'].includes(fieldType)) {`